### ISSUE TITLE
webkit crash with some nvidia drivers

### ISSUE THREAD

**Issue by col-one** [dlenski/gp-saml-gui#85]
Webkit freezes with error : `/nv_gbm.c:99: GBM-DRV error (nv_gbm_bo_create): DRM_IOCTL_NVIDIA_GEM_ALLOC_NVKMS_MEMORY failed ` after the first opening windows. 

Workaround : 
remove webkit's WebGL settings, not sure it's necessary to enable WebGL (on by default) for a such tool. 
`gp_saml_gui.py:59`:
```python
settings.set_enable_webgl(False)
```

Thanks for your work.

**Comment by dlenski** [comment]
> remove webkit's WebGL settings, not sure it's necessary to enable WebGL (on by default) for a such tool.

Agreed, not sure why it's enabled by default. Let's leave this open and see if others encounter this problem as well.
